server/server_test.go

```diff
@@ -1322,12 +1324,28 @@ func startTestServer(config string) error {
 	if err != nil {
 		return err
 	}
+
 	go func() {
 		err := server.Start()
 		if err != nil {
 			return
 		}
 	}()
+
+	retry := 3
```

Comment:
Inlining things like n * time.Seconds and so on is a common practice, go check yourself https://github.com/golang/go/search?utf8=%E2%9C%93&q=sleep&type=